includes/data.php

```diff
@@ -1524,20 +1535,34 @@ function pods_mb_substr( $string, $start, $length = null, $encoding = null ) {
 /**
  * Evaluate tags like magic tags but through pods_v.
  *
- * @since 2.1
- *
- * @param string|array|object $tags     String to be evaluated.
- * @param bool                $sanitize Whether to sanitize.
- * @param null|mixed          $fallback The fallback value to use if not set, should already be sanitized.
+ * @param string|array|object $tags String to be evaluated.
+ * @param array               $args {
+ *     Function arguments.
+ *     @type bool       $sanitize        Whether to sanitize.
+ *     @type null|mixed $fallback        The fallback value to use if not set, should already be sanitized.
+ *     @type Pods       $pod             Pod to parse the tags through.
+ *     @type bool       $use_current_pod Whether to auto-detect the current Pod.
+ * }
  *
  * @return string
  *
+ * @since 2.1
+ *
  * @see pods_evaluate_tag
  */
-function pods_evaluate_tags( $tags, $sanitize = false, $fallback = null ) {
+function pods_evaluate_tags( $tags, $args = array() ) {
```

Comment:
:no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: 0798ce6e6917bd1fce0da922565dce2b</sub>